website/src/models/task.js

```diff
@@ -50,10 +50,14 @@ var checklist = [{
 
 var DailySchema = new Schema(
   _.defaults({
-    type: {type:String, 'default': 'daily'},
+    type: {type: String, 'default': 'daily'},
+    //TODO: Cleaner to store interval as enum instead of str?
+    frequency: {type: String, 'default': 'daily'}, // 'daily', 'weekly'
```

Comment:
Default here is set to daily, but currently, a weekly is closer to what we have as a default. That means that 3rd party apps creating dailies, intending to create tasks that behave like weeklies will instead create tasks that behave like the new dailies format.